cookbooks/aws-parallelcluster-slurm/recipes/config/config_external_slurmdbd_s3_mountpoint.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: config_head_node
+#
+# Copyright:: 2013-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+ext = platform?('ubuntu') ? "deb" : "rpm"
+subtree = arm? ? "arm64" : "x86_64"
+mount_s3_url = "https://s3.amazonaws.com/mountpoint-s3-release/latest/#{subtree}/mount-s3.#{ext}"
+download_path = "/tmp/mount-s3.#{ext}"
+
```

Comment:
We may want to add here a link to the docs

https://github.com/awslabs/mountpoint-s3

Reply:
Also in this post (updated at 6/30/2023) 

https://aws.amazon.com/blogs/storage/the-inside-story-on-mountpoint-for-amazon-s3-a-high-performance-open-source-file-client/

this is still considered an `alpha` release (not recommended in production). This is ok for the external slurm dbd but it may require a deeper evaluation if someone plans to use it in ParallelCluster.

Reply:
Done in [this commit](https://github.com/aws/aws-parallelcluster-cookbook/pull/2633/commits/2d332315f644b6b26ceaac9f410ec622a18a4617) (to be squashed when merging the PR).